src/Editor/index.js

```diff
@@ -517,12 +530,15 @@ export class Editor extends React.Component {
                                 name={'message'}
                                 value={state.inputText}
                                 onBlur={props.toggleEditor}
+                                onKeyPress={this._onKeyPress}
                                 onChangeText={this.onChange}
                                 selection={this.state.selection}
                                 selectionColor="rgba(0, 0, 0, 0.5)"
                                 onSelectionChange={this.handleSelectionChange}
                                 onContentSizeChange={this.onContentSizeChange}
                                 scrollEnabled={false}
+                                keyboardType={'email-address'}
+                                returnKeyLabel={this.state.isTrackingStarted ? 'Add mention' : undefined}
```

Comment:
If this is not working remove it. But maybe it's not working because it's too long.

Try instead with `returnKeyType="done"` which is supported in both platforms. https://facebook.github.io/react-native/docs/textinput#returnkeytype